requirements.txt

```diff
@@ -1,10 +1,10 @@
-django>=1.11,<2.2
+django>=1.11,<3.0
```

Comment:
Your research indicates `Django 1.11` should be dropped.

Reply:
Actually, I was wrong, it's `Dropped support for Python 2.6 and 3.3, and Django < 1.11`, so this will work. :smile: